client.go

```diff
@@ -285,6 +279,25 @@ func (r *marathonClient) apiCall(method, uri string, body, result interface{}) e
 	return apiErr
 }
 
+// apiRequest creates a default api request
+func (r *marathonClient) apiRequest(method, url string, reader io.Reader) (*http.Request, error) {
+	// Make the http request to Marathon
```

Comment:
"Make" could be misinterpreted as in "send" whereas it means "create" here. Maybe use the latter instead?